content/200-orm/800-more/300-upgrade-guides/200-upgrading-versions/500-upgrading-to-prisma-6.mdx

```diff
@@ -190,8 +205,42 @@ In an effort to improve compatibility between Prisma and new modern JavaScript r
 
 Prisma v6 replaces the usage of [`Buffer`](https://nodejs.org/api/buffer.html)  with [`Uint8Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array) to represent fields of type `Bytes`. Make sure to replace all your occurences of the `Buffer` type with the new `Uint8Array`.
 
+<details>
+<summary>Expand to view how to convert between <code>Buffer</code> and <code>Uint8Array</code></summary>
+
+### Conversion from Buffer to Uint8Array
+
+You can directly use the `Buffer` instance as a `Uint8Array`:
+
+```ts
+const buffer: Buffer = Buffer.from([1, 2, 3, 4]);
+const uint8Array: Uint8Array = buffer; // No conversion needed
+```
+
+If you need a copy of the data (to avoid shared memory):
+
+```ts
+const uint8ArrayCopy: Uint8Array = new Uint8Array(buffer);
+```
+
+### Conversion from Uint8Array to Buffer
+
+You can create a `Buffer` from a `Uint8Array` using `Buffer.from`:
+
+```ts
+const uint8Array: Uint8Array = new Uint8Array([1, 2, 3, 4]);
+const buffer: Buffer = Buffer.from(uint8Array);
```

Comment:
This copies the data. To share memory, you need to do `Buffer.from(uint8Array.buffer, uint8Array.byteOffset, uint8Array.byteLength)`.